dextra-components/src/stories/MapMultiples.stories.ts

```diff
@@ -0,0 +1,37 @@
+import { Meta } from "@storybook/web-components";
+import { html } from "lit";
+import "../components/marks/Geo/src/Geo.ts";
+import "../components/Plot/src/Plot.ts";
+
+export default {
+  title: "Marks",
+}
+
+export const MapMultiples = () => {
+    return html`
+      <osl-plot
+        height = 2000
+        marginLeft="50"
+        projection="albers"
+        fyInterval="10 years"
+        fyTickFormat="${(d) => `${d.getUTCFullYear()}’s`}"
+      >
+        <osl-geo
+            data="http://localhost:6006/data/nations.json" 
+        ></osl-geo>
+        <osl-geo
+            data="http://localhost:6006/data/states.json" 
+            strokeOpacity="0.5"
+        ></osl-geo>
+        <osl-geo
+            data="http://localhost:6006/data/walmarts.json"
+            fy="(d) => d.properties.date" 
```

Comment:
Given some shared ownership of the faceting and some confusion that could come up, do you think we should migrate all faceting up to the plot component?

Reply:
Yes, this is what we're pursuing here. The facet functionality is integrated within the plot component through `fyInterval="10 years"`. In the geo mark, `fy="(d) => d.properties.date"` serves the purpose of specifying the specific column or variable employed for faceting. Does that sounds fair or is there any specific results that you would like to see as a property?